app/istqb-ctal-atlas-complete-guide/istqb-ctal-atlas-complete-guide.css

```
@@ -0,0 +1,1097 @@
+
+            /* ============================================================
+   PROGRESS ITEMS
+   ============================================================ */
+            .progress-list {
+                display: flex;
+                flex-direction: column;
+                gap: 0.75rem;
+                margin: 1.5rem 0;
+            }
+            .progress-item {
+            }
```

Stylelint (17.9.0)

[error] Empty block (block-no-empty)

(block-no-empty)